scc/install-scc.md

```diff
@@ -7,6 +7,9 @@ from the Tanzu Application Platform package repository.
 Both the full and light profiles include Supply Chain Choreographer.
 For more information about profiles, see [About Tanzu Application Platform package and profiles](../about-package-profiles.md).
 
+>**Note:** The Supply Chain Choreographer is now bundled with the Cartographer Conventions.
```

Comment:
should this be the other way around? i.e. "Cartographer Conventions is now bundled with Supply Chain Choreographer"